test/reporters/multiple.js

```diff
@@ -0,0 +1,98 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var Mocha = require('../../');
+var Suite = Mocha.Suite;
+var Runner = Mocha.Runner;
+var Test = Mocha.Test;
+var should = require('should');
+
+describe('multiple reporters', function() {
+  var suite;
+  var runner;
+  var specReporter;
+  var jsonReporter;
+
+  it('should have 1 test failure', function(done) {
+    var mocha = new Mocha({
+      reporter: ['spec', 'json']
+    });
+    suite = new Suite('Multiple reporters suite', 'root');
+    runner = new Runner(suite);
+    specReporter = new mocha._reporters[0].fn(runner);
+    jsonReporter = new mocha._reporters[1].fn(runner);
+
+    var testTitle = 'json test 1';
+    var error = { message: 'oh shit' };
```

Comment:
hmm, these variables are only being used once, it actually makes it harder to read to extract them from the code below.